src/scss/components/switch/_index.scss

```diff
@@ -0,0 +1,148 @@
+@import "./config";
+
+hx-switch-control {
+  -ms-grid-column-align: start;
+  -ms-grid-columns: auto 1fr;
+  -ms-grid-rows: auto auto;
+  display: -ms-grid;
+  display: grid;
+  grid-template-areas:
+    "ctrl lbl"
+    ".    help";
+  grid-template-columns: auto 1fr;
+  grid-template-rows: auto auto;
+  justify-items: start;
+  grid-area: facade;
+  margin-top: 0.25rem; // FIXME: fix with proper inline styling
+  position: relative;
+  width: 4.25rem;
+  height: 2rem;
+
+  > input {
+    opacity: 0;
+    width: 0;
+    height: 0;
+  }
+
+  > label > hx-switch {
+    position: absolute;
+    cursor: pointer;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    -webkit-transition: 0.4s;
+    transition: 0.4s;
+    border-radius: 2.125rem;
+  }
+
+  > label > hx-switch::before {
+    position: absolute;
+    content: "";
+    height: 2rem;
+    width: 2rem;
+    -webkit-transition: 0.4s;
+    transition: 0.4s;
+    border-radius: 50%;
+  }
+}
+
+hx-switch-control > input[type="checkbox"] ~ label {
+  > hx-switch {
+    background-color: $gray-0;
+    border: 0.125rem solid $cyan-900;
+
+    &:hover {
+      @include hxSwitchControl($pseudo-state: hover,$checkBox-state: unchecked);
+    }
+  }
+
+  > hx-switch[offlabel]::after {
+    content: attr(offlabel);
+    text-transform: uppercase;
+    margin-left: 2rem;
+    color: $cyan-900;
+    line-height: 1.75rem;
+    max-width: 1.813rem;
+    display: inline-block;
+    overflow: hidden;
+    white-space: nowrap;
+    font-size: 0.875rem;
+    font-family: "Roboto Mono", monospace;
+    font-weight: 500;
+  }
+
+  > hx-switch::before {
+    background-color: $gray-0;
+    border: 0.125rem solid $cyan-900;
+    left: -0.125rem;
+    top: -0.062rem;
+  }
+}
+
+hx-switch-control > input[type="checkbox"]:checked ~ label {
+  > hx-switch {
+    background-color: $cyan-900;
+    @include hxSwitchControl($pseudo-state: focus);
```

Comment:
#### It looks like this line is adding a focus glow the default state:

<img width="1013" alt="Screen Shot 2020-07-27 at 4 21 49 PM" src="https://user-images.githubusercontent.com/10120600/88593631-75fb6b00-d025-11ea-90b6-aa8dbd31fdde.png">


#### Update to:

```scss
  > hx-switch {
    background-color: $cyan-900;
    @include hxSwitchControl($pseudo-state: null);
 
    &:focus {
      @include hxSwitchControl($pseudo-state: focus);
    }

   ...
```

Reply:
Yes, the line is adding focus. But, by adding focus selector it is not working. So, i prefer to continue with the same code.

Reply:
Ok @lalithkarikelli, let's get feedback from @araiford on this.

We may be able to publish as `beta` and iterate if needed.